src/main/resources/application.yaml

```diff
@@ -92,6 +92,10 @@ server:
   error:
     whitelabel:
       enabled: false
+sftp:
+  username: ${SFTP_USERNAME}
+  password: ${SFTP_PASSWORD}
+  upload-url: ${SFTP_UPLOAD_URL}
```

Comment:
Cool - will you add these to the Github production environment + deploy script, or I can if you'd like?

Reply:
Oh yeah, I can add them